source/adios2/toolkit/derived/parser/lexer.l

```diff
@@ -149,6 +149,7 @@ void
 adios2::detail::ASTDriver::parse (const std::string input)
 {
   adios2::detail::parser parse (*this);
+  yy_flex_debug = trace_scanning;
```

Comment:
Do we need to keep this flag in the final version?

Reply:
This allows for dynamically setting debug flags without rebuilding the parser (set in ASTDriver). If we don't want this option, we should also remove the boolean members of ASTDriver, but I would argue for leaving them for now while we extend the parser with features like magnitude(curl(x,y,z), 2)

Reply:
I may have misunderstood, this line should have been in this function from the start, it's the equivalent of the later line:
parse.set_debug_level (trace_parsing);

but for the scanner instead of the parser. This is just relaying the trace flag set in ASTDriver